src/main/java/com/stripe/model/Customer.java

```diff
@@ -274,6 +274,28 @@ public Customer delete(Map<String, Object> params, RequestOptions options)
     return request(ApiResource.RequestMethod.DELETE, url, params, Customer.class, options);
   }
 
+  /** Retrieves the list of transactions applied to a customer’s balance. */
+  public CustomerBalanceTransactionCollection balanceTransactions() throws StripeException {
+    return balanceTransactions((Map<String, Object>) null, (RequestOptions) null);
+  }
+
+  /** Retrieves the list of transactions applied to a customer’s balance. */
+  public CustomerBalanceTransactionCollection balanceTransactions(Map<String, Object> params)
+      throws StripeException {
+    return balanceTransactions(params, (RequestOptions) null);
+  }
+
+  /** Retrieves the list of transactions applied to a customer’s balance. */
+  public CustomerBalanceTransactionCollection balanceTransactions(
+      Map<String, Object> params, RequestOptions options) throws StripeException {
+    String url =
+        String.format(
+            "%s%s",
+            Stripe.getApiBase(),
+            String.format("/v1/customers/%s/customer_balance_transactions", this.getId()));
+    return requestCollection(url, params, CustomerBalanceTransactionCollection.class, options);
+  }
+
```

Comment:
Flagging that since CreditNote and BalanceTransaction are not going to be released at the same time, we will need to do 2 separate PRs.